docs/imcce/imcce.rst

```diff
@@ -0,0 +1,496 @@
+.. doctest-skip-all
+
+.. _astroquery.imcce:
+
+******************************************************************************************************************************************
+Institut de Mécanique Céleste et de Calcul des Éphémérides (IMCCE) Solar System Services (`astroquery.imcce`/astroquery.solarsystem.imcce)
+******************************************************************************************************************************************
+
+Overview
+========
+
+IMCCE provides a number of Solar System-related services, two of which are currently implemented here:
+
+* `SkyBoT <http://vo.imcce.fr/webservices/skybot/>`_: search for and
+  identify Solar System objects that are present in a given area of
+  the sky at a given time
+* `Miriade <http://vo.imcce.fr/webservices/miriade/>`_: ephemerides service
+
+Use cases for both services are detailed below.
+  
+
+SkyBot - Solar System Body Identification Service
+=================================================
+
+Cone Search
+-----------
+
+`astroquery.imcce.SkybotClass` provides an interface to the `cone search
+<http://vo.imcce.fr/webservices/skybot/?conesearch>`_ offered by SkyBoT.
+
+A simple cone search for Solar System objects in a circular field
+looks like this:
+
+.. code-block:: python
+	  
+   >>> from astroquery.imcee import Skybot
+   >>> Skybot.cone_search((0, 0), 0.1, 2451200)  # doctest: +SKIP
```

Comment:
What do these numbers mean?  I feel like these should be (required to be) quantities.  At least, the documentation should clarify what they mean.

Reply:
All examples will be changed to use `SkyCoord` and the docs will be updated.